### DISCUSSION TITLE
ΠΑΡΑΔΟΤΕΟ 2 ΣΤΕΦΑΝΙΔΗΣ ΠΟΛΥΔΩΡΟΣ Π2018139

### DISCUSSION THREAD

**Discussion by p18stef** [courses-ionio/sw#1353] [Show and tell]
Σε αυτό το σκέλος της εργασίας καλούμαστε να ολοκληρώσουμε μια άσκηση γραμμής εντολών. Πιο συγκεκριμένα επέλεξα να χρησιμοποιήσω την εφαρμογή του telegram και μέσω του ntfy να στέλνω ειδοποιήσεις στο κινητό μου σχετικά με τον καιρό της πόλης μου. Αρχικά έκανα install το ntfy  στο σύστημα linux, στην συνέχεια κατέβασα στο κινητό μου την εφαρμογή του telegram. Μέσω telegram δημιούργησα ένα bot και χρησιμοποίησα το token που μου έδωσε ώστε να εξασφαλίσω την εποικινωνία. Στην συνέχεια βρήκα έναν κώδικα σε python [ Εδώ ](polis.py) σχετικά με την διεργασία που ήθελα εγώ να εκτελεί. Στην συνέχεια θα παρουσιάσω δύο scrennshots τα οποία σχετίζονται με τις ειδοποιήσεις που έρχονται στο κινητό αλλά και μία μικρή συνομιλία με το bot του τύπου "hi"

 - [ΑΝΑΦΟΡΑ](https://github.com/p18stef/sw/edit/2018139/projects/2018139/README.md)
 
- [asciinema neofetch](https://asciinema.org/a/o4XS6TphLhfJjSjCEi3hnc77g)

- [asciinema telegram](https://asciinema.org/a/TPSk9GpfopDdQi5kPnKySd7dO)

**Comment by aggelos2000430** [comment]
Αγαπητέ συνάδελφε για τις δραστηριότητες της δεύτερης εβδομάδας είναι αρκετή η εγκατάσταση ενός συστήματος που χρησιμοποιεί systemd ώς το init system του, καθώς και η καταγραφή των κανόνων συνεργασίας στην ομάδα σας. Μπορείς να ανατρέξεις και στα ακόλουθα discussions:
- [#1270](https://github.com/courses-ionio/sw/discussions/1270#discussioncomment-5110579)
-  [#1234](https://github.com/courses-ionio/sw/discussions/1234#discussioncomment-5055547)
